x-pack/plugins/viz_editor/common/select_builder.ts

```diff
@@ -22,6 +22,14 @@ const operationTemplate: OperationTemplate = {
       interval: 'auto',
     },
   },
+  window: {
+    operation: 'window',
+    argument: {
+      field: '',
+      windowFunction: 'unweightedAvg',
+      windowSize: 5,
+    },
+  },
```

Comment:
Not sure if you got this to work, but I checked it out locally and window functions didn't seem to do anything for time series. Am I doing something wrong?

Reply:
No it doesn't, this is currently not implemented in our to_esaggs mapping (and I'm not 100% sure whether it's even implemented completely in esaggs itself). It's just there as an illustration for a more complex operation configuration to explore how this can be integrated. cc @chrisdavies

Reply:
Okay, glad I wasn't missing something